widget/entry.go

```diff
@@ -1118,6 +1132,17 @@ func NewEntry() *Entry {
 	return e
 }
 
+// Bind will Bind this widget to the given DataItem
```

Comment:
These two functions were placed between the constructors "func NewEntry" and "func NewMultiLineEntry", seems like the wrong place - maybe move up to join the other "func (e *Entry)"?